src/style.css

```diff
@@ -0,0 +1,19 @@
+.first {
+  height: 100px;
```

Comment:
move common styles to common class

Reply:
I've put extra tags in <head> and moved common styles to common class. Thank you!